pkgs/tools/security/imhex/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, mbedtls
+, gtk3
+, pkgconfig
+, capstone
+, libGLU
+, glfw3
+, file
+, python3
+, curl
+, xlibs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ImHex";
+  version = "1.11.1";
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
```

Comment:
Do we have all the package that are vendored in nixpkgs? If so we can remove this.